README.md

```diff
@@ -4,7 +4,7 @@ BuzzwordCRM is a sample application that demonstrates how to build applications
 
 ## Running the app locally
 
-1. Register as a Calendly developer (if you haven't already) by logging into your Calendly account and [completing this form](https://docs.google.com/forms/d/e/1FAIpQLSc9ltoPU9I_yyQt1gWLm6fa0xMhpPWm-mL_vfPfeilC_s1vTA/viewform) (note: this app assumes that your redirect uri will be http://localhost:3000/oauth/callback). We’ll process your request within 1 day and email you when it’s time to complete the setup.
+1. Register as a Calendly developer (if you haven't already) by logging into your Calendly account and [Create a developer account](https://developer.calendly.com/create-a-developer-account) (note: this app assumes that your redirect uri will be http://localhost:3000/oauth/callback). We’ll process your request within 1 day and email you when it’s time to complete the setup.
```

Comment:
Should we remove the parts about "logging into your Calendly account" and  "We’ll process your request within 1 day and email you when it’s time to complete the setup."

```suggestion
1. Register as a Calendly developer (if you haven't already) [Create a developer account](https://developer.calendly.com/create-a-developer-account) (note: this app assumes that your redirect uri will be http://localhost:3000/oauth/callback).
```

Reply:
Yes, I think we should remove mention of logging in and the approval process, as Andrew mentions above!

Reply:
Good find! Removing it